### PR TITLE
fix: use folder material design icon

### PR DESCRIPTION
**Pull request by skjnldsv**
| Before | After |
|:---------:|:------:|
|localhost_8080_index php_apps_files_files_209_dir=_ (1)|localhost_8080_index php_apps_files_files_209_dir=_|
|localhost_8080_index php_apps_dashboard_ (1)|localhost_8080_index php_apps_dashboard_|

Fix https://github.com/nextcloud/recommendations/issues/748

### CODE REVIEW

**Review by szaimen — APPROVED** [review]
LGTM but didnt test

File: src/components/RecommendedFile.vue [hunk 1]
```diff
@@ -11,7 +11,9 @@
 		@click.prevent="navigate"
 		@keyup.enter.prevent="navigate">
 		<div class="thumbnail"
-			:style="{ 'background-image': 'url(' + previewUrl + ')' }" />
+			:style="{ 'background-image': isFolder ? '' : 'url(' + previewUrl + ')' }">
+			<FolderIcon v-if="isFolder" />
+		</div>
```
Comment (susnux) [inline src/components/RecommendedFile.vue:16]: Does this not work? Might get rid of the `:deep` then.
```vue
<FolderIcon v-if="isFolder" class="thumbnail" />
<div v-else class="thumbnail" :style="{ 'background-image': 'url(' + previewUrl + ')' }" />
```

Comment (skjnldsv) [reply src/components/RecommendedFile.vue:16]: Works too sure :+1:

Comment (skjnldsv) [reply src/components/RecommendedFile.vue]: Done @susnux !!

**Review by marcoambrosini — APPROVED** [review]

**Review by Pytal — APPROVED** [review]

**Comment by skjnldsv** [comment]
https://github.com/nextcloud/recommendations/pull/749
https://github.com/nextcloud/recommendations/pull/750

**Pull request merged**